bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -204,12 +208,27 @@ impl Client {
             tokio::sync::RwLock<Option<SessionVerificationController>>,
         > = Default::default();
         let controller = session_verification_controller.clone();
+        sdk_client.add_event_handler(move |event: AnyToDeviceEvent| async move {
```

Comment:
Since we only care about `m.key.verification.request` events in this handler, perhaps it makes sense to listen to only those types of events.

Like we do here: https://github.com/matrix-org/matrix-rust-sdk/blob/4c46e42201da5d7b1871b05764a35930d578dba6/examples/emoji_verification/src/main.rs#L126-L136.

Reply:
Good point, will do 👍